snuba/datasets/__init__.py

```diff
@@ -0,0 +1,40 @@
+from snuba import settings
+
+
+class DataSet(object):
+    """
+    A DataSet defines the complete set of data sources, schemas, and
+    transformations that are required to:
+        - Consume, transform, and insert data payloads from Kafka into Clickhouse.
+        - Define how Snuba API queries are transformed into SQL.
+
+    This is the the initial boilerplate. schema and processor will come.
+    """
+
+    def __init__(self, *args, **kwargs):
+        self.SCHEMA = None
+        self.PROD = False
+
+    def default_conditions(self, body):
+        """
+        Return a list of the default conditions that should be applied to all
+        queries on this dataset.
+        """
+        return []
+
+
+DATASETS_IMPL = {}
+
+
+def get_dataset(name):
+    assert name in settings.DATASETS
```

Comment:
That is correct. The original implementation was a little different (it would not have worked in tests so I had to change it). There the assertion was useful. Will remove.